diagnostic.md

```diff
@@ -36,13 +41,18 @@ Now, we want to get a **single donut** from the server. Write the `curl` request
 you'd use to retrieve a **single donut**, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request GET 'http://www.example.com/donuts/$25'
 ```
 
 Write an AJAX request to retrieve a **single donut** from the server.
 
 ```js
-let getDonut = /* your answer here */;
+let getDonut = function (event) {
+  return $.ajax({
+  url:'http://www.example.com/donuts' + 25,
+  method: 'GET',
+  });
+};
```

Comment:
Again, with `event`. For line 52, if `25` is hard coded into your function, this function can only ever get the 25th donut. Here's a better solution:
```
let getDonut = function (donutID) {
  return $.ajax({
    url: 'http://www.example.com/donuts/' + donutID,
    method: 'GET',
  });
};
```